src/tests/row.rs

```diff
@@ -11,6 +11,72 @@ mk_tests_default! {
     use std::str;
     use rand::{distributions::{Alphanumeric, Standard}, Rng};
 
+    #[cfg(feature = "linking")]
+    #[cfg(not(feature = "embedded_tests"))]
+    #[test]
+    fn column_with_charset_none() -> Result<(), FbError> {
+        use crate::{ConnectionBuilder, charset::ISO_8859_1};
+
+        // This test reproduce the of using a diferent
+        // charset of column in insert.
+        // When we read the data, even using the UTF8,
+        // we got an "invalid utf-8" error.
```

Comment:
Just a comment, why the "even using UTF8"? The data was `ISO8859_1` encoded, so
it makes sense to not be able to read it as `UTF8`.

Reply:
My mistake. Fixed.